addons/website/static/tests/builder/custom_tab/builder_components/builder_range.test.js

```diff
@@ -0,0 +1,47 @@
+import { expect, test } from "@odoo/hoot";
+import { animationFrame, click, waitFor } from "@odoo/hoot-dom";
+import { xml } from "@odoo/owl";
+import { contains } from "@web/../tests/web_test_helpers";
+import { delay } from "@web/core/utils/concurrency";
+import {
+    addActionOption,
+    addOption,
+    defineWebsiteModels,
+    setupWebsiteBuilder,
+} from "../../website_helpers";
+
+defineWebsiteModels();
+
+test("should commit changes", async () => {
+    addActionOption({
+        customAction: {
+            getValue: ({ editingElement }) => editingElement.innerHTML,
+            apply: ({ editingElement, value }) => {
+                expect.step(`customAction ${value}`);
+                editingElement.innerHTML = value;
```

Comment:
In terms of API: where is `value` coming from and what makes it safe to set as `innerHTML`?

If it's an unrealistic case for a test, wouldn't the test have the same coverage if this was set as `textContent` instead, avoiding the ci flag?

And if the HTML is important, then likely `value` should have been marked up somewhere (where it was validated to be safe), in which case `innerHTML` could be replaced by `setElementContent`.